### PR TITLE
[MVP4] Ship Microsoft Search to v1.0

### PR DESCRIPTION
**Pull request by lanhongvp**
remove the beta part for shipping to 1.0.

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -1800,10 +1800,10 @@
     },
     {
         "id": "81fb8b64-cb56-4de3-9cad-fb1e56d5cdba",
-        "category": "Search (beta)",
+        "category": "Search",
         "method": "POST",
         "humanName": "page search results",
-        "requestUrl": "/beta/search/query",
+        "requestUrl": "/v1.0/search/query",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/resources/search-api-overview?view=graph-rest-beta#page-search-results",
```
Comment (lanhongvp) [inline sample-queries/sample-queries.json:1807]: Will update the doc link after the doc PR merge

**Review by irvinesunday — APPROVED** [review]
👍🏽

**Review by thewahome — APPROVED** [review]
LGTM

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -1817,9 +1817,9 @@
     },
     {
         "id": "dac56b00-97a5-4072-a3fe-56750819f28d",
-        "category": "Search (beta)",
+        "category": "Search",
         "method": "POST",
-        "humanName": "sort search results",
+        "humanName": "(beta) sort search results",
```
Comment (thewahome) [inline sample-queries/sample-queries.json:1822]: Why do we specify the (beta) here?

**Pull request merged**